packages/react-utilities/src/utils/shouldPreventDefaultOnKeyDown.ts

```diff
@@ -7,6 +7,11 @@ import * as React from 'react';
  * Useful for situations where a keydown needs to be transformed to a click event
  */
 export function shouldPreventDefaultOnKeyDown(e: KeyboardEvent | React.KeyboardEvent) {
+  console.log(e.defaultPrevented);
```

Comment:
```suggestion
```

No thanks 👍

Reply:
yeah removed already just before this comment 😋